ui/StatusQ/src/StatusQ/Core/LocaleUtils.qml

```diff
@@ -312,4 +312,8 @@ QtObject {
     function getMonthYear(value) {
         return formatDate(value, "MMM yyyy")
     }
+
+    function getDayName(value) {
+        return Qt.locale().dayName(value.getDay())
```

Comment:
You should use `standaloneDayName()` otherwise this will use the possessive form